api-migration.md

```diff
@@ -0,0 +1,28 @@
+# Migrating to new APIs
+
+This document covers migrating your code when a nodejs-cloudant release has
+breaking API changes. Each section covers migrating from one particular
+version to another. The section titles state the specific versions between
+which the change was made; often one can migrate from earlier versions than
+the one specified on the left of the title to the new version specified
+on the right.
+
+Feel free to file bugs if the instructions don't work for you; we'll use
```

Comment:
This sounds like you don't have much confidence in the instructions. I'm not sure of a better wording but customers generally don't hold back in reporting bugs when things break. Do we need this para?

Reply:
This was all stolen from CTD. I think you're right. I'll drop it.